### ISSUE TITLE
Automate updating of Mozilla CA and Google CT lists

### ISSUE THREAD

**Issue by ckreibich** [zeek/zeek#3645]
We recently observed that manually updating the certificate lists built into the distribution (as done in #3621) isn't great. Several options for improvement: 
* Minimally, build scripting to automate the manual steps required right now,
* Optionally, consider invoking such automation via CI so it's not something the release maintainer needs to do every time,
* Ideally, users who need latest and greatest would be able to do so easily — perhaps via a standalone package that deploys the latest at install time and overrides the distribution's baked-in lists.

@0xxon I'm assigning to you as per recent discussion, thanks!

**Comment by timwoj** [comment]
Should these updates be done as part of the release process?

**Comment by 0xxon** [comment]
Yes - and if it’s not in the release process that’s probably my fault :)

**Comment by timwoj** [comment]
> Should these updates be done as part of the release process?

To answer my own question here, it's already described in the release manager docs. Having a script that does it instead of manual steps would be useful.

**Comment by timwoj** [comment]
@0xxon I'm going to reassign this to me.

**Comment by timwoj** [comment]
Mozilla keeps the certdata file on their Mercurial server now at https://hg-edge.mozilla.org/mozilla-central/raw-file/default/security/nss/lib/ckfw/builtins/certdata.txt. This is identical to the version in the latest NSS release. It would be easier to modify the gen-mozilla-ca-list.rb script to download this than it would be to pull the latest tarball and extract the file from it.

**Comment by timwoj** [comment]
The two PRs above automate this to where it's just one script that needs to be run during release management. I don't think this needs to happen in any more automated fashion than that. I've also updated the release manager doc about it.

**Comment by 0xxon** [comment]
Wohoo :)

Thank you very much.
